src/WebJobs.Script.WebHost/BreakingChangeAnalysis/ChangeAnalysisService.cs

```diff
@@ -128,6 +129,8 @@ public IEnumerable<AssemblyReport> LogBreakingChangeReport(CancellationToken can
             cancellationToken.ThrowIfCancellationRequested();
 
             var generator = new BreakingChangeReportGenerator();
+            var removedAssembliesString = FileUtility.ReadResourceString($"{ScriptConstants.ResourcePath}.AssembliesRemoved.txt");
```

Comment:
nit: json file would allow to add additional context / properties to the file. `AssemblyRemovedManifest.json`

Reply:
thanks! I am trying to repro few errors. If change this file to JSON if needed